packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -583,8 +583,12 @@ public void onCaptureCompleted(
         };
 
     try {
-      captureSession.stopRepeating();
-      captureSession.abortCaptures();
+      if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.O) {
+        captureSession.stopRepeating();
+      } else {
+        captureSession.stopRepeating();
+        captureSession.abortCaptures();
```

Comment:
There needs to be a comment explaining why this is conditional.

Reply:
I can confirm that this line causes troubles on some old Huawei/Xiaomi devices.